util/Logger.js

```diff
@@ -6,34 +6,33 @@ const moment = require("moment");
 
 exports.log = (content, type = "log") => {
   const timestamp = `[${moment().format("YYYY-MM-DD HH:mm:ss")}]:`;
-    switch (type) {
-      case "log": {
-        return console.log(`${timestamp} ${chalk.bgBlue(type.toUpperCase())} ${content} `);
-      }
-      case "warn": {
-        return console.log(`${timestamp} ${chalk.black.bgYellow(type.toUpperCase())} ${content} `);
-      }
-      case "error": {
-        return console.log(`${timestamp} ${chalk.bgRed(type.toUpperCase())} ${content} `);
-      }
-      case "debug": {
-        return console.log(`${timestamp} ${chalk.green(type.toUpperCase())} ${content} `);
-      }
-      case "cmd": {
-        return console.log(`${timestamp} ${chalk.black.bgWhite(type.toUpperCase())} ${content}`);
-      }
-      case "ready": {
-        return console.log(`${timestamp} ${chalk.black.bgGreen(type.toUpperCase())} ${content}`);
-      } 
-      default: throw new TypeError("Logger type must be either warn, debug, log, ready, cmd or error.");
+  switch (type) {
+    case "log": {
+      return console.log(`${timestamp} ${chalk.bgBlue(type.toUpperCase())} ${content} `);
     }
-  } 
-} 
+    case "warn": {
+      return console.warn(`${timestamp} ${chalk.black.bgYellow(type.toUpperCase())} ${content} `);
+    }
+    case "error": {
+      return console.error(`${timestamp} ${chalk.bgRed(type.toUpperCase())} ${content} `);
+    }
+    case "debug": {
+      return console.debug(`${timestamp} ${chalk.green(type.toUpperCase())} ${content} `);
```

Comment:
`console.debug` is an alias of `console.log`, and it seems to be pretty new, even not existent in some versions. (Last time I used, in Node.js 7.x, that did not exist). It also appears to be documented in [nodejs docs](https://nodejs.org/api/console.html#console_console_debug_data_args) (although it's 'since' is not documented, the version of node.js which implemented that is unknown for now) but not in [mdn docs](https://developer.mozilla.org/en-US/docs/Web/API/Console/debug). (Although it exist in Chrome 62).

For safety, could you revert this line to `console.log`, please?